iidm/iidm-extensions/src/main/resources/xsd/injectionObservability.xsd

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2021, RTE (http://www.rte-france.com)
+    This Source Code Form is subject to the terms of the Mozilla Public
+    License, v. 2.0. If a copy of the MPL was not distributed with this
+    file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+-->
+<xs:schema version="1.0"
+           xmlns:xs="http://www.w3.org/2001/XMLSchema"
+           targetNamespace="http://www.itesla_project.eu/schema/iidm/ext/injection_observability/1_0"
+           elementFormDefault="qualified">
+
+    <xs:element name="injectionObservability">
+        <xs:complexType>
+            <xs:sequence>
+                <xs:element name="qualityP">
+                    <xs:complexType>
+                        <xs:attribute name="standardDeviation" use="required" type="xs:float"/>
+                        <xs:attribute name="redundant" use="required" type="xs:boolean"/>
+                    </xs:complexType>
+                </xs:element>
+                <xs:element name="qualityQ">
+                    <xs:complexType>
+                        <xs:attribute name="standardDeviation" use="required" type="xs:float"/>
+                        <xs:attribute name="redundant" use="required" type="xs:boolean"/>
+                    </xs:complexType>
+                </xs:element>
+                <xs:element name="qualityV">
+                    <xs:complexType>
+                        <xs:attribute name="standardDeviation" use="required" type="xs:float"/>
+                        <xs:attribute name="redundant" use="required" type="xs:boolean"/>
+                    </xs:complexType>
+                </xs:element>
+            </xs:sequence>
+            <xs:attribute name="observable" use="required" type="xs:boolean"/>
```

Comment:
This one should be optional. For quality, @sylvlecl should specify if both standardDeviation and redundancy are required.